src/System.Private.CoreLib/shared/System/GCMemoryInfo.cs

```diff
@@ -53,4 +53,42 @@ namespace System
             FragmentedBytes = fragmentedBytes;
         }
     }
+
+    // TODO: own file
+    public readonly struct GCConfigInfo
+    {
+        public bool AllowVeryLargeObjects { get; }
```

Comment:
As public API, it would be more interesting to expose maximum size of array of given type instead. It is hard-coded as constant in number of places currently that is less than ideal.

Reply:
The API is internal now -- just needed it for testing.

Reply:
It needs comments to say so.